src/views/list.ts

```diff
@@ -0,0 +1,65 @@
+/*!
+ * © 2019 Atypon Systems LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  getListType,
+  JatsStyleType,
+  ManuscriptNode,
+} from '@manuscripts/transform'
+
+import { getActualAttrs } from '../lib/track-changes-utils'
+import { BaseNodeProps } from './base_node_view'
+import BlockView from './block_view'
+import { createNodeOrElementView } from './creators'
+import { EditableBlock } from './editable_block'
+export class ListView<
+  PropsType extends BaseNodeProps
+> extends BlockView<PropsType> {
+  public elementType = 'ul'
+
+  public updateContents = () => {
+    const actualAttrs = getActualAttrs(this.node)
+    if (this.contentDOM) {
+      const type = actualAttrs.listStyleType as JatsStyleType
+      this.contentDOM.style.listStyleType = getListType(type).style
```

Comment:
Not particularly important but since you run that string (JatsStyleType) inside the getListType through a switch with 'default' statement anyway, you may as well make this function accept string and you won't have to import JatsStyleType.